service.go

```diff
@@ -110,30 +112,35 @@ func (hs *k8sHealthcheckService) watchServices() {
 			hs.services.m[service.name] = service
 			hs.services.Unlock()
 
-			infoLogger.Printf("Service with name %s added or updated.", service.name)
+			log.Infof("Service with name %s added or updated.", service.name)
 		case watch.Deleted:
 			k8sService := msg.Object.(*core.Service)
 			hs.services.Lock()
 			delete(hs.services.m, k8sService.Name)
 			hs.services.Unlock()
-			infoLogger.Printf("Service with name %s has been removed", k8sService.Name)
+			log.Infof("Service with name %s has been removed", k8sService.Name)
 		default:
-			errorLogger.Print("Error received on watch services. Channel may be full")
+			log.Error("Error received on watch services. Channel may be full")
 		}
 	}
 
-	infoLogger.Print("Services watching terminated. Reconnecting...")
+	log.Info("Services watching terminated. Reconnecting...")
 	hs.watchServices()
 }
 
 func initializeHealthCheckService() *k8sHealthcheckService {
 	httpClient := &http.Client{
-		Timeout: 12 * time.Second,
```

Comment:
We should keep this at 12s - we expect services to respond within 10s, and this timeout is there to ensure the page loads in a reasonable amount of time and it doesn't get hung up on a misbehaving service.